AlfiyaSiddique/TastyTrails
# Feature chat app

## What does this PR do?

Added chat functionality using sockets
Displays last time of the messages (eg: 1 hour ago etc)
Displays user avatar as well to distinguish messages

Fixes #2 
@AlfiyaSiddique , kindly review

https://drive.google.com/file/d/1hPuEonRq0Jq0Q7km4txaLOX11oc2y8To/view?usp=sharing

## How should this be tested?




- Create a new chat for new users
-  Add message and send it out
-  Paralelly login to a new user and try chatting, you'll see realtime sync between the messages

## Checklist



### Required
